src/main/java/com/elixr/poc/rest/response/CommonResponse.java

```diff
@@ -1,9 +1,11 @@
 package com.elixr.poc.rest.response;
 
+import lombok.Builder;
 import lombok.Data;
 
 @Data
-public class DeleteResponse {
+@Builder
+public class CommonResponse {
     private boolean success;
     private String errorMessage;
```

Comment:
Why do you need this field here? You have a CommonErrorMessage class with errorMessage field?

Reply:
Updated.